Arrowgene.Ddon.Database/Files/Database/Script/schema_sqlite.sql

```diff
@@ -51,6 +51,7 @@ CREATE TABLE IF NOT EXISTS ddon_character
     "hide_equip_lantern_pawn"    BOOLEAN                           NOT NULL,
     "arisen_profile_share_range" SMALLINT                          NOT NULL,
     "fav_warp_slot_num"          INTEGER                           NOT NULL,
+    "online_status"              SMALLINT,
```

Comment:
Should it be stored in the database? I think it's safe to assume that if a player is offline the only status it can have is offline

Reply:
my issue was that the only way i know how to get a characters online status was with Database.SelectCharacter, and since the field was not stored in the DB it was always null. there is also an away status that gets set too

Reply:
the Server.ClientLookup.GetClientByCharacterId idea worked well so i removed the column